pom.xml

```diff
@@ -192,6 +192,28 @@
         <version>1.2.17</version>
         <scope>provided</scope><!-- by log4j-over-slf4j -->
       </dependency>
+
+      <!-- Recommended versions of Mockito and PoweMock for build with Java 11 (JENKINS-55098) -->
+      <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-core</artifactId>
+        <version>2.23.4</version>
+      </dependency>
+      <dependency>
+        <groupId>org.powermock</groupId>
+        <artifactId>powermock-module-junit4</artifactId>
+        <version>2.0.0-RC.4</version>
+      </dependency>
+      <dependency>
+        <groupId>org.powermock</groupId>
+        <artifactId>powermock-api-mockito2</artifactId>
+        <version>2.0.0-RC.4</version>
+      </dependency>
+      <dependency>
+        <groupId>org.objenesis</groupId>
+        <artifactId>objenesis</artifactId>
+        <version>2.6</version>
```

Comment:
why not latest `3.0.1`?

Reply:
Because Mockito and PoweMock use 2.6 now. I preferred to not try updating the the major release

Reply:
OK, we'll see how it goes.